Updates to code contribution guide (#4551)
magento/devdocs: _includes/contributor/labels.md

All [contributions to DevDocs](https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md) receive the following labels:

> * `New topic`- New topic submissions for content that has never existed on DevDocs such as tutorials, references, instructions, and so on

Reviewer: ```suggestion
* `New topic`- New topic submissions for content that has never existed in DevDocs, such as tutorials, references, instructions, etc.
```

Reviewer 2: Add "detailed descriptions of functionality" or similar.

Reviewer 3: we changed from using etc... no latin abbreviations

Reviewer: Hmm... "and so on" just doesn't seem like nomenclature we'd want to include. 🤷‍♀

Reviewer 3: You're welcome to ask the one who changed it.

Reviewer 3: Personally, I prefer "...and other relevant content."

> * `Major update` - Significant updates to existing content, such as a new section or examples with sample code

Reviewer 2: I don't think adding a section should necessarily count as a major update. We've had some inconsequential sections added that, by this definition, count as major.

Reviewer 4: @keharper Can you suggest a different description

Reviewer 2: Significant detailed new content that affects one or more existing topics.

> * `Technical` - Updates to the code or processes that alter the technical content of the document such as code snippets, reference documentation, parameter names and values, and so on

Reviewer: ```suggestion
* `Technical` - Updates to the code or processes that alter the technical content of the document, such as code snippets, reference documentation, parameter names and values, etc.
```
